deepface/detectors/YunetWrapper.py

```diff
@@ -0,0 +1,65 @@
+import cv2
+import os
+import gdown
+from deepface.detectors import FaceDetector
+from deepface.commons import functions
+
+
+def build_model():
+    url = "https://github.com/opencv/opencv_zoo/raw/main/models/face_detection_yunet/face_detection_yunet_2023mar.onnx"
+    file_name = "face_detection_yunet_2023mar.onnx"
+    home = functions.get_deepface_home()
+    if os.path.isfile(home + f"/.deepface/weights/{file_name}") is False:
+        print(f"{file_name} will be downloaded...")
+        output = home + f"/.deepface/weights/{file_name}"
+        gdown.download(url, output, quiet=False)
+    face_detector = cv2.FaceDetectorYN_create(
+        home + f"/.deepface/weights/{file_name}", "", (0, 0)
+    )
+    return face_detector
+
+
+def detect_face(detector, image, align=True, score_threshold=0.9):
+    # FaceDetector.detect_faces does not support score_threshold parameter.
+    # We can set it via environment variable.
+    score_threshold = os.environ.get("yunet_score_threshold", score_threshold)
+    resp = []
+    detected_face = None
+    img_region = [0, 0, image.shape[1], image.shape[0]]
+    faces = []
+    height, width = image.shape[0], image.shape[1]
+    # resize image if it is too large (Yunet fails to detect faces on large input sometimes)
+    # I picked 640 as a threshold because it is the default value of max_size in Yunet.
+    if height > 640 or width > 640:
+        r = 640.0 / max(height, width)
```

Comment:
If you resize the image, you need to restore it after detection. Coordinates should be updated, too.

Reply:
Fixed in latest commit. Tested on some large images and appeared work. Please check.